specs/createHeadlessBrowser.js

```diff
@@ -10,6 +10,7 @@ export default function createHeadlessBrowser(testOptions) {
   const options = testOptions || {};
   const systemLanguage = options.systemLanguage || 'en';
   const proxyConsole = options.proxyConsole || false;
+  const initialStorage = options.initialStorage || {};
```

Comment:
Please use `Object.create(null)` and `Object.assign` instead of plain object.
You can read more about `Object.create(null)` and `{}` here http://stackoverflow.com/questions/15518328/creating-js-object-with-object-createnull